atomic_reactor/utils/cachi2.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2024 Red Hat, Inc
+# All rights reserved.
+#
+# This software may be modified and distributed under the terms
+# of the BSD license. See the LICENSE file for details.
+
+"""
+Utils to help to integrate with cachi2 CLI tool
+"""
+
+from typing import Any, Dict
+
+
+def remote_source_to_cachi2(remote_source: Dict[str, Any]) -> Dict[str, Any]:
+    """Converts remote source into cachi2 expected params.
+
+    Remote sources were orignally designed for cachito. Cachi2 is not a direct
+    fork but has lot of similarities.
+    However, some parameters must be updated to be compatible with cachi2."""
```

Comment:
Should we include more information about the parameters? Or are we expecting that we will always remove only the "include-git-dir" and "remove-unsafe-symlinks" flags?

Reply:
only these 2 flags as it's job for OSBS in future, and also that pkg manager Adam mentioned in https://github.com/containerbuildsystem/atomic-reactor/pull/2092#discussion_r1681040180